x-pack/plugins/siem/public/management/pages/index.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { PolicyContainer } from './policy';
+import {
+  MANAGEMENT_ROUTING_ENDPOINTS_PATH,
+  MANAGEMENT_ROUTING_POLICIES_PATH,
+  MANAGEMENT_ROUTING_ROOT_PATH,
+} from '../common/constants';
+import { ManagementPageView } from '../components/management_page_view';
+
+const TmpEndpoints = () => {
+  return (
+    <ManagementPageView viewType="list" headerLeft="Test">
+      <h1>{'Endpoints will go here'}</h1>
+      <SpyRoute />
+    </ManagementPageView>
+  );
+};
+
+export const ManagementContainer = memo(() => {
```

Comment:
Container loads routes for the sub-sections of this area. Those sub-sections then defined their own `<Route ...` for additional routes under them.  I think this makes each area more self-contained, but am open to bring them all into this component here

Reply:
@paul-tavares does keeping the areas more self contained help us keep them behind a flag?

Reply:
yes, it does.
But, since currently only Endpoint "stuff" is under management, I think the "turning off" management if the config flag is set to true, will be done higher up in the chain - I noticed `SIemNavTab` definition (`x-pack/plugins/siem/public/app/home/home_navigations.tsx`) has a `disabled: boolean` property - perhaps that is the location where we "hide" management.